styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -122,6 +122,7 @@
 
   public static final int SCHEDULER_TICK_INTERVAL_SECONDS = 2;
   public static final int TRIGGER_MANAGER_TICK_INTERVAL_SECONDS = 1;
+  public static final int CLEANER_TICK_INTERVAL_SECONDS = 30;
```

Comment:
As we discussed, 30s is too much for polling. I think we could be very happy already to have it run every half an hour or even more. xD